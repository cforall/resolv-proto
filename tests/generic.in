#int foo-i #ptr<#int>
#float foo-f #ptr<#float>

#ptr<T> addr T

%%

foo( #ptr<#int> )
foo( #ptr<#float> )
foo( addr(#int) )
foo( addr(#float) )