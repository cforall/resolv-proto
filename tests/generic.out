
[#int / (0,0,0,0,0)] foo-i( #ptr<#int> )

[#float / (0,0,0,0,0)] foo-f( #ptr<#float> )

[#int / (0,2,1,-1,0)] foo-i( addr( #int ) )

[#float / (0,2,1,-1,0)] foo-f( addr( #float ) )
